java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_2/Controller.java

```diff
@@ -0,0 +1,172 @@
+                "\nTo create Robot with values initialization enter \"2\"");
+
+        while (true) {
+            Scanner scanner = new Scanner(System.in);
+            int inputValue;
+            if (scanner.hasNextInt()) {
+                inputValue = scanner.nextInt();
+                if (inputValue == 1) {
+                    robot = new Robot();
+                    break;
+                } else if (inputValue == 2) {
+                    System.out.println("Enter Room's width as integer value.");
+                    int roomWidth = getValue();
+                    roomWidth--;
+
+                    System.out.println("Enter Room's height as integer value.");
+                    int roomHeight = getValue();
+                    roomHeight--;
+
+                    System.out.println("Enter Robot's default direction as String value (south, west, north or east).");
+                    String direction;
+                    outer:
+                    while (true) {
+                        Scanner input = new Scanner(System.in);
+                        String line = input.nextLine().toUpperCase();
+
+                        switch (line) {
+                            case "NORTH" -> {
+                                direction = "NORTH";
+                                break outer;
+                            }
+                            case "SOUTH" -> {
+                                direction = "SOUTH";
+                                break outer;
+                            }
+                            case "EAST" -> {
+                                direction = "EAST";
+                                break outer;
+                            }
+                            case "WEST" -> {
+                                direction = "WEST";
+                                break outer;
+                            }
+                            default -> {
+                                System.out.println("Define correct direction.");
+                                continue outer;
+                            }
+                        }
+                    }
+
+                    System.out.println("Enter Robot's starting width as integer value.");
+                    int robotWidth = getValue();
+
+                    System.out.println("Enter Robot's starting height as integer value.");
+                    int robotHeight = getValue();
+
+                    robot = new Robot(roomWidth, roomHeight, direction, robotWidth, robotHeight);
+                    break;
+                } else {
+                    System.out.println("Enter \"1\" or \"2\"");
+                    continue;
+                }
+            } else {
+                System.out.println("Enter integer value");
+                continue;
+            }
+        }
+        System.out.println("You create Room of " + (robot.getRoom().width() + 1) + " titles height and of " +
```

Comment:
This can be defined in the `toString` method

Reply:
Move this in the "toString" method.